docs/antctl.md

```diff
@@ -422,6 +418,42 @@ results:
     action: Delivered
 ```
 
+To start a live-traffic traceflow, add `--live-traffic` (or `-L`) flag. Add
+`--dropped-only` flag to indicate only the packet dropped by a NetworkPolicy
+should be captured in the live-traffic traceflow. A live-traffic traceflow
+just requires one of `--source` and `--destination` arguments to be specified,
+and at least one of them must be a Pod.
+
+`--flow` (or `-f`) argument can be used to specify the traceflow packet headers
+with the [ovs-ofctl](http://www.openvswitch.org//support/dist-docs/ovs-ofctl.8.txt)
+flow syntax. The supported flow fields include: IP family (`ipv6` to indicate an
+IPv6 packet), IP protocol (`icmp`, `icmpv6`, `tcp`, `udp`), source and
+destination ports (`tcp_src`, `tcp_dst`, `udp_src`, `udp_dst`), and TCP flags
+(`tcp_flags`).
+
+By default, the command will wait for the traceflow succeeds or fails, until
```

Comment:
the command will wait for the traceflow to succeed or fail, or until a timeout

Reply:
Updated to "the command will wait for the traceflow to succeed or fail, or timeout".